packages/core/src/components/forms/_input-group.scss

```diff
@@ -35,6 +36,8 @@ Styleguide input-group
 $input-button-height: $pt-button-height - 6px !default;
 // 5px space between regular button and large input
 $input-button-height-large: $pt-button-height !default;
+// 1px space between regular button and small input
+$input-button-height-small: $pt-button-height - 8px !default;
```

Comment:
refactor these variables to reference the `$pt-button-height-<size>` vars above please. they should be the same.

Reply:
Like this: 
```scss
// 3px space between small button and regular input
$input-button-height: $pt-button-height-small !default;
// 5px space between regular button and large input
$input-button-height-large: $pt-button-height !default;
// 1px space between regular button and small input
$input-button-height-small: $pt-button-height-smaller !default;
```

?

Reply:
@giladgray